src/index.json

```diff
@@ -984,6 +984,39 @@
                     "version": "0.1.2"
                 }
             }
+        ],
+        "aladdin": [
+            {
+                "filename": "aladdin-0.0.1-py3-none-any.whl",
+                "sha256Digest": "6c70f04fc807ed7efb36d67a9b421413c81e425a3c4159b1d26652cdf63f8729",
+                "downloadUrl": "https://files.pythonhosted.org/packages/0c/06/78ddfe634d2af7b35a4931b7a17aea9f026c202316b18f861f462857a2fc/aladdin-0.0.1-py3-none-any.whl",
```

Comment:
CI gives 404s for this url. It needs to be publicly available.

Reply:
I thought my whl file will be uploaded to that URL as part of the merge. I'll fix it.